components/RecipeList/index.js

```diff
@@ -14,18 +33,41 @@ function RefreshButton({ onRefetch }) {
 }
 
 export default function RecipeListRandom({ recipes }) {
+  const [selectedMealType, setSelectedMealType] = useState("");
+
   const { data, error, isLoading, mutate } = useSWR(
-    `/api/spoonacular/recipes/random?number=20`
+    `/api/spoonacular/recipes/random?number=20&tags=${selectedMealType}`
   );
 
   if (error) return <p>failed to load</p>;
   if (isLoading) return <p>loading...</p>;
 
+  const handleMealType = (event) => {
+    setSelectedMealType(event.target.value);
+  };
+
   return (
     <>
       <Heading>New Ideas</Heading>
-
       <StyledContainer>
+        <StyledSection>
+          <StyledLabel htmlFor="mealType">Food cravings</StyledLabel>
+          <StyledOption>
```

Comment:
Since `StyledOption` is in fact a `section`, its only purpose is styling, right? If so, you can remove it and pass its styling rules to the `select` element as a wrapper for the actual `option`s.